environments/staging-aws-test/inventory.ini

```diff
@@ -86,8 +86,15 @@ remote_postgresql
 
 [redis]
 # ElastiCache
-staging-test-redis.ck3xwy.0001.use1.cache.amazonaws.com:6379
+{{ redis-staging-test }}:6379
```

Comment:
This is technically wrong, but also in a fake env, so it's fine to ignore this. If I need to fix it, I'll fix it later, but we'll likely destroy this env before needing to use it again.